lib/ffprober.rb

```diff
@@ -8,26 +8,15 @@
 require_relative 'ffprober/subtitle_stream'
 require_relative 'ffprober/chapter'
 require_relative 'ffprober/ffprobe_version'
+require_relative 'ffprober/wrapper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.